bin/metrics-docker-info.rb

```diff
@@ -0,0 +1,92 @@
+#! /usr/bin/env ruby
+#
+#   metrics-docker-stats
+#
+# DESCRIPTION:
+#
+# Supports the stats feature of the docker remote api ( docker server 1.5 and newer )
+# Supports connecting to docker remote API over Unix socket or TCP
+#
+#
+# OUTPUT:
+#   metric-data
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#
+# USAGE:
+#   Gather stats from all containers on a host using socket:
+#   metrics-docker-stats.rb -p unix -H /var/run/docker.sock
+#
+#   Gather stats from all containers on a host using TCP:
+#   metrics-docker-stats.rb -p http -H localhost:2375
+#
+#   Gather stats from a specific container using socket:
+#   metrics-docker-stats.rb -p unix -H /var/run/docker.sock -c 5bf1b82382eb
+#
+#   See metrics-docker-stats.rb --help for full usage flags
+#
+# NOTES:
+#
+# LICENSE:
+#   Copyright 2015 Paul Czarkowski. Github @paulczar
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/metric/cli'
+require 'socket'
+require 'net_http_unix'
+require 'json'
+
+class DockerStatsMetrics < Sensu::Plugin::Metric::CLI::Graphite
+  option :scheme,
+         description: 'Metric naming scheme, text to prepend to metric',
+         short: '-s SCHEME',
+         long: '--scheme SCHEME',
+         default: "#{Socket.gethostname}.docker"
+
+  option :docker_host,
+         description: 'Docker socket to connect. TCP: "host:port" or Unix: "/path/to/docker.sock" (default: "127.0.0.1:2375")',
+         short: '-H DOCKER_HOST',
+         long: '--docker-host DOCKER_HOST',
+         default: '127.0.0.1:2375'
+
+  option :docker_protocol,
+         description: 'http or unix',
+         short: '-p PROTOCOL',
+         long: '--protocol PROTOCOL',
+         default: 'http'
```

Comment:
Not a problem, we're using unix socket as well, I just copied same argument than for other metrics scripts (which uses http as default), I prefer to have unix socket as default IMHO.